framework/resources/Datasources/table_stats/analyze_tables.sql

```diff
@@ -0,0 +1,19 @@
+alter session set `store.parquet.reader.int96_as_timestamp`=true;
```

Comment:
Can you please change this to set `store.parquet.reader.int96_as_timestamp`=true;

Reply:
@Agirish Will that not change it at a system wide level if we did "set store..."?, Do we need the option to be set at system level, that will affect all other tests.

Reply:
@kfaraaz , please refer https://drill.apache.org/docs/set/

Reply:
@Agirish thanks, that makes sense to me know.